content/birds/jonny.rst

```diff
@@ -2,11 +2,12 @@ public: yes
 bird: jonny
 role: 'co-founder and front-end lead'
 adr:
-  - city: 'Goshen'
-    state: 'IN'
+  - city: Goshen
+    state: IN
 social:
-  - twitter: 'jgerigmeyer'
-    github: 'jgerigmeyer'
+  - twitter: jgerigmeyer
+    github: jgerigmeyer
+facebook: jgerigmeyer
```

Comment:
Yeah, the reason I didn't nest it with the other `social` keys is that personally I'm not sure I want my Facebook page linked from my bio, but I don't mind having it linked (via the `og` tag) when a blog post is shared on Facebook. So as this PR stands now, it's being treated differently than the other social links.